src/twisted/conch/ssh/keys.py

```diff
@@ -198,6 +240,16 @@ def _fromString_BLOB(cls, blob):
                     )
                 ).public_key(default_backend())
             )
+        elif keyType in [b'ecdsa-sha2-' + curve for curve in list(_curveTable.keys())]:
+            x, y, rest = common.getMP(rest, 2)
+            newKey =  cls(
+                ec.EllipticCurvePublicNumbers(
+                    x, y,
+                    _curveTable[keyType.split(b'-')[2]]
+                    ).public_key(default_backend())
+                )
+            newKey.ecKeyName = keyType
```

Comment:
You're adding a new attribute here, `ecKeyName`, but it's not documented.  Better would be if it's not public!  This is not information that anyone should be using externally, as far as I can tell.